portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,17 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Gábor Fuisz - 2020 STEP Internship Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div id="header">
+      <img src="images/selfie.jpg" id="selfie">
+      <div id="header-right">
+        <h1 id="introduction">My name is Gábor Fuisz</h1>
+        <p id="explanation">And this is my portfolio for the 2020 Google STEP internship.</p>
+      </div>
```

Comment:
I recommend using classes over ids for everything that is not accessed via JavaScript or where there is a possibility that multiple elements of that type could be necessary eventually.
E.g. an `explanation` is something generic that should be reusable everywhere so it's not a unique element on the whole page and therefore shouldn't be an `id` but rather a `class`.